core/classifiers/base.py

```diff
@@ -36,6 +36,14 @@ class BaseClassifier(object):
     def __init__(self):
         pass
 
+    @property
+    @abc.abstractproperty
+    def version(self):
+        """Version of the classifier algorithm. Version of algorithm is matched
```

Comment:
Nit on wording: ``The version  of the algorithm is matched...``

Reply:
Done